Source/WebCore/page/Quirks.cpp

```diff
@@ -198,8 +198,7 @@ bool Quirks::shouldDisableContentChangeObserver() const
     if (!needsQuirks())
         return false;
 
-    auto& topDocument = m_document->topDocument();
-    if (isDomain("youtube.com"_s) && (topDocument.url().path().startsWithIgnoringASCIICase("/results"_s) || topDocument.url().path().startsWithIgnoringASCIICase("/watch"_s)))
+    if (isDomain("youtube.com"_s))
```

Comment:
It should return be:
```
return isDomain("youtube.com"_s);
```

Reply:
done. Thanks for the review. 
I also added the missing comment as a reference.